backend/src/main/java/com/eshka/service/impl/TeamServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.eshka.service.impl;
+
+import com.eshka.entity.Team;
+import com.eshka.exception.TeamNotFoundException;
+import com.eshka.repository.TeamRepository;
+import com.eshka.service.TeamService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class TeamServiceImpl implements TeamService {
+    private final TeamRepository teamRepository;
+
+    @Override
+    public Team findById(long id) {
+        return teamRepository.findById(id).orElseThrow();
```

Comment:
Here you should specify the type of exception. In your case, it's `TeamNotFoundException`. Small example:
`.orElseThrow(() -> new TeamNotFoundException("team not found"))` instead of simple `.orElseThrow()`.